src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+     class="text-form"
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+    >
```

Comment:
```suggestion
    <form
      class="text-form"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```
3 point in checklist